### PR TITLE
feat: handle automatic shim dl

### PR DESCRIPTION
**Pull request by swarnimarun**
Replaces PR #421 

fixes CED-782

### CODE REVIEW

File: cmd/helper.go [hunk 1]
```diff
@@ -73,6 +73,7 @@ var helperCmd = &cobra.Command{
 
 		port, _ := cmd.Flags().GetUint32(portFlag)
 		startChroot, _ := cmd.Flags().GetBool("start-chroot")
+    startChroot = startChroot || setupHost
```
Comment (swarnimarun) [inline cmd/helper.go:76]:
```suggestion
        startChroot = startChroot || setupHost
```

File: cmd/helper.go [hunk 2]
```diff
@@ -73,6 +73,7 @@ var helperCmd = &cobra.Command{
 
 		port, _ := cmd.Flags().GetUint32(portFlag)
 		startChroot, _ := cmd.Flags().GetBool("start-chroot")
+        startChroot = startChroot || setupHost
```
Comment (nravic) [inline cmd/helper.go:76]: is startChroot used at all anymore? The func can probably just go

**Review by nravic — APPROVED** [review]
lgtm, added small question

File: cmd/helper.go (hunk 2 above)
Comment (swarnimarun) [reply cmd/helper.go:76]: It's used only in a helper function to do a single call with chroot. It's never passed as a flag anywhere in k8s workflow.

I will make tickets today to clean up k8s helper part of the project.

**Pull request merged**